life/updated/03-testimonies.adoc

```diff
@@ -47,25 +47,25 @@ John Edmonds, Michael Turpin, Moses West, Michael Turpin, Jr. Edmond Neale, Jame
 [.blurb]
 === John Thornton`'s account of his neighbor, Elizabeth Stirredge.
 
-I had an intimate acquaintance with her;
-and as the Lord had given her a measure of the holy Spirit,
+I had an intimate acquaintance with her,
+and as the Lord had given her a measure of the holy Spirit
```

Comment:
I generally capitalize Holy when it comes before Spirit.... since it is generally considered a proper name (even though it isn't really)